docs/news.rst

```diff
@@ -5,6 +5,11 @@ Release notes
 
 .. _release-1.6.0:
 
+Scrapy 1.6.1 (2019-MM-DD)
```

Comment:
This is unlikely to be in 1.6.1: first, it is a new feature (which may be not suited for a point release), and second, we don't know if we'll have 1.6.1, or go straight into 1.7. 

Could you please remove a specific version here? You can either replace it with "Unreleased (TBA)", or just remove changelog changes from this pull request.

Reply:
Restored.